db/connect.js

```diff
@@ -0,0 +1,20 @@
+const mongoose = require('mongoose')
+
+// http://mongoosejs.com/docs/promises.html
+mongoose.Promise = global.Promise
```

Comment:
@m4v15 I use native promises, I think especially if you do not have experience with promises, native is totes fine